exercises/practice/bob/.approaches/config.json

```diff
@@ -0,0 +1,35 @@
+{
+  "introduction": {
+    "authors": ["bobahop", "erikschierboom"]
```

Comment:
Casing might not matter, but for consistency I would use `"ErikSchierboom"` here.

Reply:
Ah, that makes sense. If it's used elsewhere, I will go ahead and script up a change across all of Exercism if necessary.

Reply:
Casing indeed doesn't matter. Apparently I've been inconsistent myself, sorry for the confusion.